runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java

```diff
@@ -307,34 +307,52 @@ public void onMenuEntryAdded(MenuEntryAdded menuEntryAdded)
 		// widget ticking and prior to our client tick event. This is because drag start
 		// is what builds the context menu row which is what the eventual click will use
 
-		// Deposit- op 2 is the current withdraw amount 1/5/10/x
-		if (shiftModifier && menuEntryAdded.getType() == MenuAction.CC_OP.getId() && menuEntryAdded.getIdentifier() == 2
-			&& config.swapBankOp() && menuEntryAdded.getOption().startsWith("Deposit-"))
+		// Swap to shift-click deposit behavior
+		// Deposit-, CC_OP, and id 2 is the current withdraw amount 1/5/10/x
```

Comment:
The original comment here is correct. Deposit op 2 is the current withdraw amount. We just have the op called "identifier". The "op" also implies CC_OP.